.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in the repo.
+* @it-at-m/refarch-admin @it-at-m/refarch-maintainer
```

Comment:
@devtobi is there a reason for mentioning admins and maintainers here?

René mentioned he is getting many notifications without really needing his attention, so only maintainers would be better fitting in my opinion.

Reply:
We can definitely change that. I wanted to admins be able to approve as well. Problem is, when we only set maintainers here, admins (even they have more rights than maintainers) will not be able to approve an MR in terms of code ownership.
If admins won't actively participate in development, we can of course remove those from the CODEOWNER file. However the CODE_OWNERS file itself should only be approvable by admins.

Reply:
As far as i know admins are mostly also added to the maintainer group.
I create a MR for that